src/dmd/typesem.d

```diff
@@ -1206,6 +1206,10 @@ extern(C++) Type typeSemantic(Type t, Loc loc, Scope* sc)
             argsc.stc = 0; // don't inherit storage class
             argsc.protection = Prot(Prot.Kind.public_);
             argsc.func = null;
+            Loc tloc = Loc();
+            auto mockFunc = new FuncDeclaration(tloc, tloc, Id.__tmpFunc, sc.stc, new TypeFunction(ParameterList(), Type.tvoid, LINK.d, sc.stc));
```

Comment:
Giving the function declaration and identifier will add it to the symbol table which might make it accessible to user code.